tests/integration/container/utils/rds_test_utility.py

```diff
@@ -392,18 +392,18 @@ def _get_aurora_topology_sql(self, engine: DatabaseEngine) -> str:
             return ("SELECT SERVER_ID, SESSION_ID FROM information_schema.replica_host_status "
                     "ORDER BY IF(SESSION_ID = 'MASTER_SESSION_ID', 0, 1)")
         elif engine == DatabaseEngine.PG:
-            return ("SELECT SERVER_ID, SESSION_ID FROM aurora_replica_status() "
-                    "ORDER BY CASE WHEN SESSION_ID = 'MASTER_SESSION_ID' THEN 0 ELSE 1 END")
+            return ("SELECT SERVER_ID, SESSION_ID FROM pg_catalog.aurora_replica_status() "
+                    "ORDER BY CASE WHEN SESSION_ID OPERATOR(pg_catalog.=) 'MASTER_SESSION_ID' THEN 0 ELSE 1 END")
         else:
             raise UnsupportedOperationError(engine.value)
 
     def _get_multi_az_topology_sql(self, engine: DatabaseEngine, writer_id) -> str:
         if engine == DatabaseEngine.MYSQL:
-            return f"SELECT id, endpoint, port FROM mysql.rds_topology ORDER BY id='{writer_id}' DESC"
+            return f"SELECT id, endpoint, port FROM mysql.rds_topology ORDER BY id = '{writer_id}' DESC"
```

Comment:
~~I think this can be qualified as `operator(pg_catalog.=)` as well~~
Nevermind I realized it is for mysql